### PR TITLE
[zh]sync chinese trans for feature-gate

### PR DESCRIPTION
**Pull request by zhaojizhuang**
[zh]sync chinese trans for feature-gate

### CODE REVIEW

File: content/zh/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -588,47 +652,64 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
 - `KubeletPodResources`: Enable the kubelet's pod resources grpc endpoint.
-   See [Support Device Monitoring](https://git.k8s.io/community/keps/sig-node/compute-device-assignment.md) for more details.
-- `LegacyNodeRoleBehavior`: When disabled, legacy behavior in service load balancers and node disruption will ignore the `node-role.kubernetes.io/master` label in favor of the feature-specific labels.
+   See [Support Device Monitoring](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/compute-device-assignment.md) for more details.
+- `LegacyNodeRoleBehavior`: When disabled, legacy behavior in service load balancers and node disruption will ignore the `node-role.kubernetes.io/master` label in favor of the feature-specific labels provided by `NodeDisruptionExclusion` and `ServiceNodeExclusion`.
 -->
 
 - `HyperVContainer`：为 Windows 容器启用[Hyper-V 隔离](https://docs.microsoft.com/en-us/virtualization/windowscontainers/manage-containers/hyperv-container)。
 - `HPAScaleToZero`：使用自定义指标或外部指标时，可将 `HorizontalPodAutoscaler` 资源的 `minReplicas` 设置为 0。
+- `ImmutableEphemeralVolumes`: 允许将单个 Secret 和 ConfigMap 标记为不可变的，以提高安全性和性能。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/feature-gates.md:696]:
```suggestion
- `ImmutableEphemeralVolumes`: 允许将各个 Secret 和 ConfigMap 标记为不可变更的，以提高安全性和性能。
```

File: content/zh/docs/reference/command-line-tools-reference/feature-gates.md [hunk 2]
```diff
@@ -588,47 +652,64 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
 - `MountContainers`: Enable using utility containers on host as the volume mounter.
 - `MountPropagation`: Enable sharing volume mounted by one container to other containers or pods.
   For more details, please see [mount propagation](/docs/concepts/storage/volumes/#mount-propagation).
 - `NodeDisruptionExclusion`: Enable use of the node label `node.kubernetes.io/exclude-disruption` which prevents nodes from being evacuated during zone failures.
 -->
 
-- `LocalStorageCapacityIsolation`：启用[本地临时存储](/docs/concepts/configuration/manage-compute-resources-container/)的消耗，以及 [emptyDir 卷](/docs/concepts/storage/volumes/#emptydir) 的 `sizeLimit` 属性。
-- `LocalStorageCapacityIsolationFSQuotaMonitoring`：如果为[本地临时存储](/docs/concepts/configuration/manage-compute-resources-container/)启用了 `LocalStorageCapacityIsolation`，并且 [emptyDir 卷](/docs/concepts/storage/volumes/#emptydir) 的后备文件系统支持项目配额，并且启用了这些配额，请使用项目配额来监视 [emptyDir 卷](/docs/concepts/storage/volumes/#emptydir)的存储消耗而不是遍历文件系统，以此获得更好的性能和准确性。
+- `LocalStorageCapacityIsolation`：启用[本地临时存储](/docs/concepts/configuration/manage-resources-containers/)的消耗，以及 [emptyDir 卷](/docs/concepts/storage/volumes/#emptydir) 的 `sizeLimit` 属性。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/feature-gates.md:711]:
```suggestion
- `LocalStorageCapacityIsolation`：允许使用[本地临时存储](/zh/docs/concepts/configuration/manage-resources-containers/)以及 [emptyDir 卷](/zh/docs/concepts/storage/volumes/#emptydir) 的 `sizeLimit` 属性。
```

File: content/zh/docs/reference/command-line-tools-reference/feature-gates.md [hunk 3]
```diff
@@ -588,47 +652,64 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
+- `LocalStorageCapacityIsolationFSQuotaMonitoring`：如果为[本地临时存储](/docs/concepts/configuration/manage-resources-containers/)启用了 `LocalStorageCapacityIsolation`，并且 [emptyDir 卷](/docs/concepts/storage/volumes/#emptydir) 的后备文件系统支持项目配额，并且启用了这些配额，请使用项目配额来监视 [emptyDir 卷](/docs/concepts/storage/volumes/#emptydir)的存储消耗而不是遍历文件系统，以此获得更好的性能和准确性。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/feature-gates.md:712]: use Chinese url with prefix '/zh'.

File: content/zh/docs/reference/command-line-tools-reference/feature-gates.md [hunk 4]
```diff
@@ -648,7 +730,8 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `NodeLease`：启用新的租赁 API 以报告节点心跳，可用作节点运行状况信号。
 - `NonPreemptingPriority`：为 PriorityClass 和 Pod 启用 NonPreempting 选项。
 - `PersistentLocalVolumes`：在 Pod 中启用 “本地” 卷类型的使用。如果请求 “本地” 卷，则必须指定 Pod 亲和力。
-- `PodOverhead`：启用 [PodOverhead](/docs/concepts/configuration/pod-overhead/) 特性以解决 Pod 开销。
+- `PodDisruptionBudget`: 启用 [PodDisruptionBudget](/docs/tasks/run-application/configure-pdb/) 特性.
+- `PodOverhead`：启用 [PodOverhead](/docs/concepts/scheduling-eviction/pod-overhead/) 特性以解决 Pod 开销。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/feature-gates.md:734]:
```suggestion
- `PodOverhead`：启用 [PodOverhead](/zh/docs/concepts/scheduling-eviction/pod-overhead/) 特性以考虑 Pod 开销。
```

File: content/zh/docs/reference/command-line-tools-reference/feature-gates.md [hunk 5]
```diff
@@ -671,10 +754,9 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `ProcMountType`：启用对容器的 ProcMountType 的控制。
 - `PVCProtection`：启用防止任何 Pod 仍使用 PersistentVolumeClaim(PVC) 删除的特性。可以在[此处](/docs/tasks/administer-cluster/storage-object-in-use-protection/)中找到更多详细信息。
 - `QOSReserved`：允许在 QoS 级别进行资源预留，以防止处于较低 QoS 级别的 Pod 突发进入处于较高 QoS 级别的请求资源（仅适用于内存）。
-- `ResourceLimitsPriorityFunction`：启用 scheduler 优先级特性，该特性将最低可能得 1 分配给至少满足输入 Pod 的 cpu 和内存限制之一的节点，目的是打破得分相同的节点之间的联系。
+- `ResourceLimitsPriorityFunction` （ *已弃用* ）：启用 scheduler 优先级特性，该特性将最低可能得 1 分配给至少满足输入 Pod 的 cpu 和内存限制之一的节点，目的是打破得分相同的节点之间的联系。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/feature-gates.md:757]:
```suggestion
- `ResourceLimitsPriorityFunction` （ *已弃用* ）：启用某调度器优先级函数，该函数将
  最低得分 1 指派给至少满足输入 Pod 的 cpu 和内存限制之一的节点，目的是打破得分相同的
  节点之间的关联。
```

File: content/zh/docs/reference/command-line-tools-reference/feature-gates.md [hunk 6]
```diff
@@ -694,22 +775,28 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
   A node is eligible for exclusion if labelled with "`alpha.service-controller.kubernetes.io/exclude-balancer`" key or `node.kubernetes.io/exclude-from-external-load-balancers`.
-- `ServiceTopology`: Enable service to route traffic based upon the Node topology of the cluster. See [ServiceTopology](https://kubernetes.io/docs/concepts/services-networking/service-topology/) for more details.
+- `ServiceTopology`: Enable service to route traffic based upon the Node topology of the cluster. See [ServiceTopology](/docs/concepts/services-networking/service-topology/) for more details.
+- `SetHostnameAsFQDN`: Enable the ability of setting Fully Qualified Domain Name(FQDN) as hostname of pod. See [Pod's `setHostnameAsFQDN` field](/docs/concepts/services-networking/dns-pod-service/#pod-sethostnameasfqdn-field).
 - `StartupProbe`: Enable the [startup](/docs/concepts/workloads/pods/pod-lifecycle/#when-should-you-use-a-startup-probe) probe in the kubelet.
 - `StorageObjectInUseProtection`: Postpone the deletion of PersistentVolume or
   PersistentVolumeClaim objects if they are still being used.
 -->
 
 - `SCTPSupport`：在 “服务”、“端点”、“NetworkPolicy” 和 “Pod” 定义中，将 SCTP 用作 “协议” 值。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/feature-gates.md:792]:
```suggestion
- `SCTPSupport`：在 Service、Endpoints、NetworkPolicy 和 Pod 定义中，允许将 _SCTP_ 用作 `protocol` 值。
```

File: content/zh/docs/reference/command-line-tools-reference/feature-gates.md [hunk 7]
```diff
@@ -694,22 +775,28 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
 - `ServerSideApply`：在 API 服务器上启用[服务器端应用（SSA）](/docs/reference/using-api/api-concepts/#server-side-apply) 路径。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/feature-gates.md:793]:
```suggestion
- `ServerSideApply`：在 API 服务器上启用[服务器端应用（SSA）](/zh/docs/reference/using-api/server-side-apply/) 路径。
```

File: content/zh/docs/reference/command-line-tools-reference/feature-gates.md [hunk 8]
```diff
@@ -694,22 +775,28 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
+- `ServiceAccountIssuerDiscovery`: 在 API 服务器中为服务帐户发行者启用 OIDC 发现端点（发行者和 JWKS URL）。 详情请参见 [为 Pod 配置服务账户](/docs/tasks/configure-pod-container/configure-service-account/#service-account-issuer-discovery) 。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/feature-gates.md:794]:
```suggestion
- `ServiceAccountIssuerDiscovery`: 在 API 服务器中为服务帐户颁发者启用 OIDC 发现端点
  （颁发者和 JWKS URL）。 详情请参见 [为 Pod 配置服务账户](/zh/docs/tasks/configure-pod-container/configure-service-account/#service-account-issuer-discovery) 。
```

File: content/zh/docs/reference/command-line-tools-reference/feature-gates.md [hunk 9]
```diff
@@ -694,22 +775,28 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
+- `ServiceAppProtocol`: 在 服务 和 端点 上启用 “ AppProtocol” 字段。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/feature-gates.md:795]:
```suggestion
- `ServiceAppProtocol`：为 Service 和 Endpoints 启用 `AppProtocol` 字段。
```

File: content/zh/docs/reference/command-line-tools-reference/feature-gates.md [hunk 10]
```diff
@@ -694,22 +775,28 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
 - `ServiceLoadBalancerFinalizer`：为服务负载均衡启用终结器保护。
 - `ServiceNodeExclusion`：启用从云提供商创建的负载均衡中排除节点。如果节点标记有 `alpha.service-controller.kubernetes.io/exclude-balancer` 键或 `node.kubernetes.io/exclude-from-external-load-balancers`，则可以排除节点。
 - `ServiceTopology`: 启用服务拓扑可以让一个服务基于集群的节点拓扑进行流量路由。有关更多详细信息，请参见[Service 拓扑](https://kubernetes.io/zh/docs/concepts/services-networking/service-topology/)
+- `SetHostnameAsFQDN`: 启用将完全合格域名 （FQDN）设置为 Pod 主机名的功能。请参见 [给 Pod 设置 `setHostnameAsFQDN` 字段](/docs/concepts/services-networking/dns-pod-service/#pod-sethostnameasfqdn-field).
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/feature-gates.md:799]:
```suggestion
- `SetHostnameAsFQDN`: 启用将全限定域名 （FQDN）设置为 Pod 主机名的功能。请参见 [给 Pod 设置 `setHostnameAsFQDN` 字段](/zh/docs/concepts/services-networking/dns-pod-service/#pod-sethostnameasfqdn-field).
```

File: content/zh/docs/reference/command-line-tools-reference/feature-gates.md [hunk 11]
```diff
@@ -743,7 +830,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
 -->
 
 - `TaintBasedEvictions`：根据节点上的污点和 Pod 上的容忍度启用从节点驱逐 Pod 的特性。有关更多详细信息，请参见[污点和容忍度](/docs/concepts/configuration/taint-and-toleration/)。
-- `TaintNodesByCondition`：根据[节点条件](/docs/concepts/configuration/taint-and-toleration/)启用自动在节点标记污点。
+- `TaintNodesByCondition`：根据[节点条件](/docs/concepts/scheduling-eviction/taint-and-toleration/)启用自动在节点标记污点。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/feature-gates.md:833]: use '/zh'

File: content/zh/docs/reference/command-line-tools-reference/feature-gates.md [hunk 12]
```diff
@@ -766,13 +853,17 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
+- `WindowsRunAsUserName` : Enable support for running applications in Windows containers with as a non-default user.
+  See [Configuring RunAsUserName](/docs/tasks/configure-pod-container/configure-runasusername) for more details.
 - `WinDSR`: Allows kube-proxy to create DSR loadbalancers for Windows.
 - `WinOverlay`: Allows kube-proxy to run in overlay mode for Windows.
 -->
 
 - `VolumeSubpathEnvExpansion`：启用 `subPathExpr` 字段用于将环境变量扩展为 `subPath`。
 - `WatchBookmark`：启用对监测 bookmark 事件的支持。
 - `WindowsGMSA`：允许将 GMSA 凭据规范从 Pod 传递到容器运行时。
+- `WindowsRunAsUserName` : 提供在 Windows 容器中运行应用程序时对非默认用户的支持。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/feature-gates.md:865]:
```suggestion
- `WindowsRunAsUserName` : 提供使用非默认用户在 Windows 容器中运行应用程序的支持。
```

File: content/zh/docs/reference/command-line-tools-reference/feature-gates.md [hunk 13]
```diff
@@ -766,13 +853,17 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
+  See [Configuring RunAsUserName](/docs/tasks/configure-pod-container/configure-runasusername) for more details.
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/feature-gates.md:866]: 漏译

**Comment by tengqm** [comment]
看起来此页面大多数 URL 都需要修改，请帮忙。

File: content/zh/docs/reference/command-line-tools-reference/feature-gates.md (hunk 5 above)
Comment (howieyuen) [inline content/zh/docs/reference/command-line-tools-reference/feature-gates.md:757]: `ResourceLimitsPriorityFunction` （ *已弃用* ）：启用调度器优先级函数，对于至少满足 CPU 和内存限制条件之一的节点，此函数至少分配1分。这么做的目的是打破得分相同的节点之间的联系。

**Review by howieyuen — CHANGES_REQUESTED** [review]
I found a lot of EN links and some untranslated line in this page, please recheck the whole doc and make sure that is synced fully.

**Comment by zhaojizhuang** [comment]
> 看起来此页面大多数 URL 都需要修改，请帮忙。

@tengqm Done

**Comment by zhaojizhuang** [comment]
> I found a lot of EN links and some untranslated line in this page, please recheck the whole doc and make sure that is synced fully.

@howieyuen Done it ,Thanks

File: content/zh/docs/reference/command-line-tools-reference/feature-gates.md [hunk 14]
```diff
@@ -440,14 +489,15 @@ Each feature gate is designed for enabling/disabling a specific feature:
 -->
 
 - `Accelerators`：使用 Docker 时启用 Nvidia GPU 支持。
-- `AdvancedAuditing`：启用[高级审查功能](/docs/tasks/debug-application-cluster/audit/#advanced-audit)。
-- `AffinityInAnnotations`（ *已弃用* ）：启用 [Pod 亲和力或反亲和力](/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity)。
+- `AdvancedAuditing`：启用[高级审查功能](/zh/docs/tasks/debug-application-cluster/audit/#advanced-audit)。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/feature-gates.md:492]: auditing -> 审计

File: content/zh/docs/reference/command-line-tools-reference/feature-gates.md [hunk 15]
```diff
@@ -463,14 +513,16 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
+- `AttachVolumeLimit`：启用卷插件用于报告可连接到节点的卷数限制。有关更多详细信息，请参见[动态卷限制](/zh/docs/concepts/storage/storage-limits/#dynamic-volume-limits)。
 - `BalanceAttachedNodeVolumes`：包括要在调度时进行平衡资源分配的节点上的卷数。scheduler 在决策时会优先考虑 CPU、内存利用率和卷数更近的节点。
-- `BlockVolume`：在 Pod 中启用原始块设备的定义和使用。有关更多详细信息，请参见[原始块卷支持](/docs/concepts/storage/persistent-volumes/#raw-block-volume-support)。
+- `BlockVolume`：在 Pod 中启用原始块设备的定义和使用。有关更多详细信息，请参见[原始块卷支持](/zh/docs/concepts/storage/persistent-volumes/#raw-block-volume-support)。
 - `BoundServiceAccountTokenVolume`：迁移 ServiceAccount 卷以使用由 ServiceAccountTokenVolumeProjection 组成的预计卷。有关更多详细信息，请参见 [Service Account Token 卷](https://git.k8s.io/community/contributors/design-proposals/storage/svcacct-token-volume-source.md)。
-- `CPUManager`：启用容器级别的 CPU 亲和力支持，有关更多详细信息，请参见 [CPU 管理策略](/docs/tasks/administer-cluster/cpu-management-policies/)。
+- `ConfigurableFSGroupPolicy`：在 Pod 中挂载卷时，允许用户为 fsGroup 配置卷访问权限和属主变更策略。请参见 [为 Pod 配置卷访问权限和属主变更策略](/zh/docs/tasks/configure-pod-container/security-context/#configure-volume-permission-and-ownership-change-policy-for-pods)。
+- `CPUManager`：启用容器级别的 CPU 亲和力支持，有关更多详细信息，请参见 [CPU 管理策略](/zh/docs/tasks/administer-cluster/cpu-management-policies/)。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/feature-gates.md:525]:
```suggestion
- `CPUManager`：启用容器级别的 CPU 亲和性支持，有关更多详细信息，
  请参见 [CPU 管理策略](/zh/docs/tasks/administer-cluster/cpu-management-policies/)。
```

File: content/zh/docs/reference/command-line-tools-reference/feature-gates.md [hunk 16]
```diff
@@ -685,31 +767,37 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
 - `StorageObjectInUseProtection`: Postpone the deletion of PersistentVolume or
   PersistentVolumeClaim objects if they are still being used.
 -->
 
-- `SCTPSupport`：在 “服务”、“端点”、“NetworkPolicy” 和 “Pod” 定义中，将 SCTP 用作 “协议” 值。
-- `ServerSideApply`：在 API 服务器上启用[服务器端应用（SSA）](/docs/reference/using-api/api-concepts/#server-side-apply) 路径。
+- `SCTPSupport`：在 Service、Endpoints、NetworkPolicy 和 Pod 定义中，允许将 _SCTP_ 用作 `protocol` 值。
+- `ServerSideApply`：在 API 服务器上启用[服务器端应用（SSA）](/zh/docs/reference/using-api/server-side-apply/) 路径。
+- `ServiceAccountIssuerDiscovery`：在 API 服务器中为服务帐户颁发者启用 OIDC 发现端点。
+   颁发者和 JWKS URL）。 详情请参见 [为 Pod 配置服务账户](/zh/docs/tasks/configure-pod-container/configure-service-account/#service-account-issuer-discovery) 。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/feature-gates.md:795]:
```suggestion
   颁发者和 JWKS URL）。 详情请参见[为 Pod 配置服务账户](/zh/docs/tasks/configure-pod-container/configure-service-account/#service-account-issuer-discovery) 。
```

File: content/zh/docs/reference/command-line-tools-reference/feature-gates.md [hunk 17]
```diff
@@ -685,31 +767,37 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
+- `ServiceAppProtocol`：为 Service 和 Endpoints 启用 `AppProtocol` 字段。
 - `ServiceLoadBalancerFinalizer`：为服务负载均衡启用终结器保护。
 - `ServiceNodeExclusion`：启用从云提供商创建的负载均衡中排除节点。如果节点标记有 `alpha.service-controller.kubernetes.io/exclude-balancer` 键或 `node.kubernetes.io/exclude-from-external-load-balancers`，则可以排除节点。
-- `ServiceTopology`: 启用服务拓扑可以让一个服务基于集群的节点拓扑进行流量路由。有关更多详细信息，请参见[Service 拓扑](https://kubernetes.io/zh/docs/concepts/services-networking/service-topology/)
+- `ServiceTopology`：启用服务拓扑可以让一个服务基于集群的节点拓扑进行流量路由。有关更多详细信息，请参见[Service 拓扑](https://kubernetes.io/zh/docs/concepts/services-networking/service-topology/)。
+- `SetHostnameAsFQDN`: 启用将全限定域名 （FQDN）设置为 Pod 主机名的功能。请参见 [给 Pod 设置 `setHostnameAsFQDN` 字段](/zh/docs/concepts/services-networking/dns-pod-service/#pod-sethostnameasfqdn-field)。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/feature-gates.md:800]:
```suggestion
- `SetHostnameAsFQDN`: 启用将全限定域名 （FQDN）设置为 Pod 主机名的功能。请参见[给 Pod 设置 `setHostnameAsFQDN` 字段](/zh/docs/concepts/services-networking/dns-pod-service/#pod-sethostnameasfqdn-field)。
```

File: content/zh/docs/reference/command-line-tools-reference/feature-gates.md [hunk 18]
```diff
@@ -685,31 +767,37 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
-- `ServerSideApply`：在 API 服务器上启用[服务器端应用（SSA）](/docs/reference/using-api/api-concepts/#server-side-apply) 路径。
+- `SCTPSupport`：在 Service、Endpoints、NetworkPolicy 和 Pod 定义中，允许将 _SCTP_ 用作 `protocol` 值。
+- `ServerSideApply`：在 API 服务器上启用[服务器端应用（SSA）](/zh/docs/reference/using-api/server-side-apply/) 路径。
+- `ServiceAccountIssuerDiscovery`：在 API 服务器中为服务帐户颁发者启用 OIDC 发现端点。
+   颁发者和 JWKS URL）。 详情请参见 [为 Pod 配置服务账户](/zh/docs/tasks/configure-pod-container/configure-service-account/#service-account-issuer-discovery) 。
+- `ServiceAppProtocol`：为 Service 和 Endpoints 启用 `AppProtocol` 字段。
 - `ServiceLoadBalancerFinalizer`：为服务负载均衡启用终结器保护。
 - `ServiceNodeExclusion`：启用从云提供商创建的负载均衡中排除节点。如果节点标记有 `alpha.service-controller.kubernetes.io/exclude-balancer` 键或 `node.kubernetes.io/exclude-from-external-load-balancers`，则可以排除节点。
-- `ServiceTopology`: 启用服务拓扑可以让一个服务基于集群的节点拓扑进行流量路由。有关更多详细信息，请参见[Service 拓扑](https://kubernetes.io/zh/docs/concepts/services-networking/service-topology/)
+- `ServiceTopology`：启用服务拓扑可以让一个服务基于集群的节点拓扑进行流量路由。有关更多详细信息，请参见[Service 拓扑](https://kubernetes.io/zh/docs/concepts/services-networking/service-topology/)。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/feature-gates.md:799]:
```suggestion
- `ServiceTopology`：启用服务拓扑可以让一个服务基于集群的节点拓扑进行流量路由。有关更多详细信息，请参见
  [Service 拓扑](/zh/docs/concepts/services-networking/service-topology/)。
```

File: content/zh/docs/reference/command-line-tools-reference/feature-gates.md [hunk 19]
```diff
@@ -742,12 +830,12 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
-- `TaintBasedEvictions`：根据节点上的污点和 Pod 上的容忍度启用从节点驱逐 Pod 的特性。有关更多详细信息，请参见[污点和容忍度](/docs/concepts/configuration/taint-and-toleration/)。
-- `TaintNodesByCondition`：根据[节点条件](/docs/concepts/configuration/taint-and-toleration/)启用自动在节点标记污点。
+- `TaintBasedEvictions`：根据节点上的污点和 Pod 上的容忍度启用从节点驱逐 Pod 的特性。有关更多详细信息，请参见[污点和容忍度](/zh/docs/concepts/configuration/taint-and-toleration/)。
+- `TaintNodesByCondition`：根据[节点条件](/zh/docs/concepts/scheduling-eviction/taint-and-toleration/)启用自动在节点标记污点。
 - `TokenRequest`：在服务帐户资源上启用 `TokenRequest` 端点。
-- `TokenRequestProjection`：启用通过 [`projected` 卷](/docs/concepts/storage/volumes/#projected) 将服务帐户令牌注入到 Pod 中的特性。
-- `TopologyManager`：启用一种机制来协调 Kubernetes 不同组件的细粒度硬件资源分配。详见 [控制节点上的拓扑管理策略](/docs/tasks/administer-cluster/topology-manager/)。
-- `TTLAfterFinished`：完成执行后，允许 [TTL 控制器](/docs/concepts/workloads/controllers/ttlafterfinished/)清理资源。
+- `TokenRequestProjection`：启用通过 [`projected` 卷](/zh/docs/concepts/storage/volumes/#projected) 将服务帐户令牌注入到 Pod 中的特性。
+- `TopologyManager`：启用一种机制来协调 Kubernetes 不同组件的细粒度硬件资源分配。详见 [控制节点上的拓扑管理策略](/zh/docs/tasks/administer-cluster/topology-manager/)。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/feature-gates.md:837]:
```suggestion
- `TopologyManager`：启用一种机制来协调 Kubernetes 不同组件的细粒度硬件资源分配。详见
  [控制节点上的拓扑管理策略](/zh/docs/tasks/administer-cluster/topology-manager/)。
```

File: content/zh/docs/reference/command-line-tools-reference/feature-gates.md [hunk 20]
```diff
@@ -766,13 +854,17 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
+  See [Configuring RunAsUserName](/docs/tasks/configure-pod-container/configure-runasusername) for more details.
 - `WinDSR`: Allows kube-proxy to create DSR loadbalancers for Windows.
 - `WinOverlay`: Allows kube-proxy to run in overlay mode for Windows.
 -->
 
 - `VolumeSubpathEnvExpansion`：启用 `subPathExpr` 字段用于将环境变量扩展为 `subPath`。
 - `WatchBookmark`：启用对监测 bookmark 事件的支持。
 - `WindowsGMSA`：允许将 GMSA 凭据规范从 Pod 传递到容器运行时。
+- `WindowsRunAsUserName`：提供使用非默认用户在 Windows 容器中运行应用程序的支持。
+  详情请参见 [Configuring RunAsUserName](/zh/docs/tasks/configure-pod-container/configure-runasusername)。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/feature-gates.md:867]:
```suggestion
  详情请参见[配置 RunAsUserName](/zh/docs/tasks/configure-pod-container/configure-runasusername)。
```

File: content/zh/docs/reference/command-line-tools-reference/feature-gates.md [hunk 21]
```diff
@@ -785,6 +877,6 @@ Each feature gate is designed for enabling/disabling a specific feature:
   the project's approach to removing features and components.
 -->
 
-* Kubernetes 的 [弃用策略](/docs/reference/using-api/deprecation-policy/) 介绍了项目已移除的特性部件和组件的方法。
+* Kubernetes 的 [弃用策略](/zh/docs/reference/using-api/deprecation-policy/) 介绍了项目已移除的特性部件和组件的方法。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/feature-gates.md:880]:
```suggestion
* Kubernetes 的[弃用策略](/zh/docs/reference/using-api/deprecation-policy/)介绍了项目处理
  已移除的特性和组件的方法。
```

**Comment by zhaojizhuang** [comment]
@tengqm Done，thanks for review

**Pull request merged**